### PR TITLE
Working db_tests

### PR DESCRIPTION
**Pull request by herpaderpaldent**
* cd into package folder
* use `composer install`
* run `vendor/bin/phpunit`

### CODE REVIEW

**Comment by herpaderpaldent** [comment]
image

File: tests/TestCase.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        $this->withFactories(__DIR__.'/../src/database/factories');
+    }
+    protected function getPackageProviders($app)
+    {
+        // ConsoleServiceProvider required to make migrations work
+        return [
+            WebServiceProvider::class,
+        ];
+    }
+    protected function getPackageAliases($app)
```
Comment (warlof) [inline tests/TestCase.php]: Since this is doing nothing, it has to be drop.

File: tests/TestCase.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        // For the facade
+        /*return [
+            'ShopifyApp' => \OhMyBrew\ShopifyApp\Facades\ShopifyApp::class,
+        ];*/
+    }
+    protected function resolveApplicationHttpKernel($app)
```
Comment (warlof) [inline tests/TestCase.php]: Since this is doing nothing, it has to be drop.

File: tests/WebTest.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+use Illuminate\Support\Facades\DB;
+
+class WebTest extends TestCase
+{
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testExample()
```
Comment (warlof) [inline tests/WebTest.php]: this must be drop as it's not testing anything.

**Pull request closed without merging**